src/tests/unit/zl.test-utils.ts

```
@@ -0,0 +1,168 @@
+      nested: this.randomObject(depth - 1),
+      array: Array.from({ length: 3 }, () => this.randomString(5))
+    };
+  }
+
+  static createTestCases<T, R>(scenarios: Omit<TestCase<T, R>, 'id'>[]): TestCase<T, R>[] {
+    return scenarios.map((scenario, index) => ({
+      ...scenario,
+      id: `test-case-${index}-${Date.now()}`
+    }));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)